# Translations template for nova.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the nova project.
#
# Translators:
# Adriana Chisco Landazábal <achisco94@gmail.com>, 2015
# Jorge Guerrero <lavidadigital@gmail.com>, 2014
# Pablo Sanchez <furybeat@gmail.com>, 2015
# Andreas Jaeger <jaegerandi@gmail.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: nova 14.0.0.0b3.dev750\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2016-08-25 06:14+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2015-10-07 05:39+0000\n"
"Last-Translator: Travis Nguyen <travisn@us.ibm.com>\n"
"Language: es\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.3\n"
"Language-Team: Spanish\n"

#, python-format
msgid ""
"%(cell)s has not been seen since %(last_seen)s and is being treated as mute."
msgstr ""
"%(cell)s no se ha observador desde %(last_seen)s y se está tratando como en "
"estado apagado."

#, python-format
msgid "%(cmd)s failed. Sleeping %(time)s seconds before retry."
msgstr "%(cmd)s ha fallado. Hibernando %(time)s segundos antes del re-ingreso."

#, python-format
msgid "%(host_state)s has not been heard from in a while"
msgstr "%(host_state)s no ha sido recibido durante un tiempo"

#, python-format
msgid "%(logprefix)s failed to load json"
msgstr "%(logprefix)s falló en cargar json"

#, python-format
msgid "%(logprefix)s file does not exist"
msgstr "El archivo %(logprefix)s no existe"

#, python-format
msgid "%(overs)s quota exceeded for %(pid)s, tried to resize instance."
msgstr ""
"%(overs)s cuota excedida para %(pid)s, se ha intentado redimensionar la "
"instancia. "

#, python-format
msgid "%(plugin)s.%(fn)s failed. Retrying call."
msgstr "%(plugin)s.%(fn)s fallido. Reintentando llamada."

#, python-format
msgid "%s failed. Not Retrying."
msgstr "%s ha fallado. No se está intentando de nuevo."

#, python-format
msgid "%s is already mounted"
msgstr "%s ya está montado "

#, python-format
msgid "%s swap image was used by instance but no back files existing!"
msgstr ""
"la instancia utilizó %s la imagen de intercambio pero ¡no existen ficheros "
"de respaldo!"

#, python-format
msgid "Address |%(address)s| is not allocated"
msgstr "La dirección |%(address)s| no está asignada"

#, python-format
msgid "Address |%(address)s| is not allocated to your project |%(project)s|"
msgstr "La dirección |%(address)s| no está asignada al proyecto |%(project)s|"

#, python-format
msgid ""
"Assigning a pci device without numa affinity toinstance %(instance)s which "
"has numa topology"
msgstr ""
"Asignando un dispositivo pci sin afinidad numa toinstance %(instance)s la "
"cual tiene topología numa"

#, python-format
msgid "Attempted auto_configure_disk failed because: %s"
msgstr "El intento de auto_configure_disk ha fallado por: %s"

#, python-format
msgid "Attempted to remove chain %s which does not exist"
msgstr "Se ha intentado eliminar la cadena %s que no existe"

#, python-format
msgid ""
"Cannot add security group %(name)s to %(instance)s since the port "
"%(port_id)s does not meet security requirements"
msgstr ""
"No se puede añadir el grupo de seguridad %(name)s a %(instance)s ya que el "
"puerto %(port_id)s no cumple los requisitos de seguridad"

#, python-format
msgid "Cannot decode JSON from %(id_path)s"
msgstr "No se puede decodificar el JSOON de %(id_path)s"

#, python-format
msgid "Cannot delete domain |%s|"
msgstr "No se puede suprimir el dominio |%s|"

#, python-format
msgid "Cannot delete entry |%s|"
msgstr "No se puede suprimir la entrada |%s|"

msgid "Cannot destroy instance, operation time out"
msgstr ""
"No se puede destruir intsancia, tiempo de espera agotado para la operación"

msgid ""
"Cannot get the number of cpu, because this function is not implemented for "
"this platform. "
msgstr ""
"No se puede obtener el número de CPU porque esta función no está "
"implementada para esta plataforma. "

msgid "Cannot purge SR with referenced VDIs"
msgstr "No se puede depurar SR con las VDI referenciadas"

#, python-format
msgid ""
"Cannot update service status on host \"%s\" due to an unexpected exception."
msgstr ""
"No se puede actualizar el estado de servicio para host \"%s\" debido a una "
"excepción inesperada."

#, python-format
msgid "Cannot update service status on host \"%s\" since it is not registered."
msgstr ""
"No se puede actualizar el estado de servicio para host \"%s\" ya que no está "
"registrado."

#, python-format
msgid "Change will make usage less than 0 for the following resources: %s"
msgstr "El cambio hará el uso menos de 0 para los siguientes recursos: %s"

msgid ""
"Cinder V1 API is deprecated as of the Juno release, and Nova is still "
"configured to use it. Enable the V2 API in Cinder and set cinder."
"catalog_info in nova.conf to use it."
msgstr ""
"Se desaprueba el uso de Cinder V1 API como una versión de Juno y Nova aún "
"está configurada para usarlo. Habilite el V2 API en Cinder y configure "
"cinder.catalog_info en nova.conf para usarlo."

#, python-format
msgid "Could not decode cpu_allocation_ratio: '%s'"
msgstr "No se puede decodificar cpu_allocation_ratio: '%s'"

#, python-format
msgid "Could not decode disk_allocation_ratio: '%s'"
msgstr "No se puede decodificar disk_allocation_ratio: '%s'"

#, python-format
msgid "Could not decode max_instances_per_host: '%s'"
msgstr "No se puede decodificar max_instances_per_host: '%s'"

#, python-format
msgid "Could not decode max_io_ops_per_host: '%s'"
msgstr "No se puede decodificar disk_allocation_ratio: '%s'"

#, python-format
msgid "Could not decode ram_allocation_ratio: '%s'"
msgstr "No se puede decodificar ram_allocation_ratio: '%s'"

msgid "Could not determine iscsi initiator name"
msgstr "No se ha podido determinar el nombre de iniciador iscsi "

#, python-format
msgid "Could not determine key: %s"
msgstr "No se ha podido determinar la clave: %s"

#, python-format
msgid ""
"Couldn't retrieve the online CPUs due to a Libvirt error: %(error)s with "
"error code: %(error_code)s"
msgstr ""
"No se pudo recuperar las CPUs en línea a causa de un error Libvirt: "
"%(error)s con código de error: %(error_code)s"

#, python-format
msgid ""
"Database inconsistency: DNS domain |%s| is registered in the Nova db but not "
"visible to either the floating or instance DNS driver. It will be ignored."
msgstr ""
"Incoherencia de base de datos: el dominio DNS |%s| está registrado en la "
"base de datos Nova pero no es visible para el controlador DNS de instancia o "
"flotante. Se ignorará."

#, python-format
msgid ""
"Destination %s already exists! Concurrent moves can lead to unexpected "
"results."
msgstr ""
"¡Ya existe el destino %s! Movimientos concurrentes pueden llevar a "
"resultados inesperados."

#, python-format
msgid "Destroy called on non-existing instance %s."
msgstr "Se ha llamado una destrucción en una instancia no existente %s"

#, python-format
msgid "Detach interface failed, port_id=%(port_id)s, reason: %(msg)s"
msgstr ""
"Ha fallado la desconexión de interfaz, port_id=%(port_id)s, razón: %(msg)s"

#, python-format
msgid "Detaching from erroneous nbd device returned error: %s"
msgstr "La desconexión del dispositivo nbd erróneo ha devuelto un error: %s"

msgid "Detaching volume from unknown instance"
msgstr "Desconectando volumen de instancia desconocida "

#, python-format
msgid ""
"Detected instance with name label '%s' which is marked as DELETED but still "
"present on host."
msgstr ""
"Se ha detectado una instancia con etiqueta de nombre '%s' la cual ha sido "
"marcada como ELIMINADA pero todavía se encuentra presente en el anfitrión."

#, python-format
msgid ""
"Detected running orphan instance: %(uuid)s (consuming %(memory_mb)s MB "
"memory)"
msgstr ""
"Se ha detectado una instancia huérfana en ejecución: %(uuid)s (consumiento "
"%(memory_mb)s MB de memoria)"

msgid "Device allocation failed after repeated retries."
msgstr ""
"Ha fallado la asignación de dispositivo después de repetidos reintentos."

#, python-format
msgid "Domain |%(domain)s| already exists, changing project to |%(project)s|."
msgstr ""
"El dominio |%(domain)s| ya existe, cambiando el proyecto a |%(project)s."

#, python-format
msgid "Domain |%(domain)s| already exists, changing zone to |%(av_zone)s|."
msgstr "El dominio |%(domain)s| ya existe, cambiando zona a |%(av_zone)s|."

msgid "During detach_interface, instance disappeared."
msgstr "Durante detach_interface, la instancia ha desaparecido."

msgid "During detach_volume, instance disappeared."
msgstr "Durante detach_volume, la instancia ha desaparecido."

#, python-format
msgid ""
"Error from libvirt during destroy. Code=%(errcode)s Error=%(e)s; attempt "
"%(attempt)d of 3"
msgstr ""
"Error de libvirt durante destrucción. Código=%(errcode)s Error=%(e)s; "
"intento %(attempt)d de 3"

#, python-format
msgid ""
"Error from libvirt while getting description of %(instance_name)s: [Error "
"Code %(error_code)s] %(ex)s"
msgstr ""
"Error de libvirt al obtener la descripción de %(instance_name)s: [Código de "
"error %(error_code)s] %(ex)s"

#, python-format
msgid "Error monitoring migration: %(ex)s"
msgstr "Error al monitorear migración: %(ex)s"

#, python-format
msgid "Error while trying to clean up image %s"
msgstr "Error al intentar limpiar imagen %s"

#, python-format
msgid "Expected to receive %(exp)s bytes, but actually %(act)s"
msgstr "Se esperaba recibir %(exp)s bytes, se han recibido %(act)s"

#, python-format
msgid ""
"Extension %(ext_name)s: Cannot extend resource %(collection)s: No such "
"resource"
msgstr ""
"Ampliación %(ext_name)s: no se puede ampliar el recurso %(collection)s: no "
"existe dicho recurso."

#, python-format
msgid "Failed to %(service)s_%(method)s: %(ex)s"
msgstr "Fallo para %(service)s_%(method)s: %(ex)s"

#, python-format
msgid "Failed to close augeas %s"
msgstr "No se ha podido cerrar augeas %s"

#, python-format
msgid "Failed to close guest handle %s"
msgstr "No se ha podido cerrar manejador de invitado %s"

msgid ""
"Failed to create block device for instance due to being over volume resource "
"quota"
msgstr ""
"No se pudo crear el dispositivo de bloque, debido a que se ha superado la "
"cuota establecida para el volumen."

#, python-format
msgid "Failed to deallocate port %(port_id)s for instance. Error: %(error)s"
msgstr ""
"Ha fallado la desasignación de puerto %(port_id)s para instancia. Error: "
"%(error)s"

#, python-format
msgid "Failed to delete port %s for instance."
msgstr "No se ha podido eliminar puerto %s para instancia."

#, python-format
msgid "Failed to delete snapshot from shelved instance (%s)."
msgstr "Fallo al remover la instantánea de la instancia almacenada (%s)."

#, python-format
msgid "Failed to delete volume: %(volume_id)s due to %(exc)s"
msgstr "No se ha podido eliminar el volumen %(volume_id)s a causa de %(exc)s"

#, python-format
msgid "Failed to disassociated floating address: %s"
msgstr "Fallo al desasociar la dirección IP flotante: %s"

#, python-format
msgid "Failed to extract instance support from %s"
msgstr "No se ha podido extraer el soporte de instancia de %s"

#, python-format
msgid "Failed to get cluster references %s"
msgstr "Fallo al obtener las referencias del cluster %s"

#, python-format
msgid "Failed to load extension %(ext_factory)s: %(exc)s"
msgstr "No se ha podido cargar la ampliación %(ext_factory)s: %(exc)s"

msgid "Failed to power off instance"
msgstr "Fallo al apagar la instancia"

msgid "Failed to resume instance"
msgstr "No se ha podido reanudar la instancia"

#, python-format
msgid "Failed to revert task state for instance. Error: %s"
msgstr "Error al volver al estado de tarea para instancia. Error: %s"

#, python-format
msgid "Failed to shutdown appliance %s"
msgstr "No se ha podido concluir el dispositivo %s"

msgid "Failed to soft reboot instance. Trying hard reboot."
msgstr ""
"Fallo al reiniciar la instancia de manera suave. Intentando reinicio duro."

#, python-format
msgid ""
"Fast cloning is only supported on default local SR of type ext. SR on this "
"system was found to be of type %s. Ignoring the cow flag."
msgstr ""
"Clonado rápido solo está soportado en SR local predeterminado de tipo ext. "
"Se ha encontrado que los SR de este sistema son de tipo %s. Ignorando el "
"identificador cow."

msgid "File injection into a boot from volume instance is not supported"
msgstr ""
"La inyección de archivo al arranque desde la instancia del volumen no está "
"soportado."

#, python-format
msgid "Flag sr_matching_filter '%s' does not respect formatting convention"
msgstr ""
"El distintivo sr_matching_filter '%s' no respeta el convenio de formato"

msgid "Flavor could not be found, skipping migration."
msgstr "El sabor no puede ser encontrado, omitiendo migración."

#, python-format
msgid ""
"Found multiple matches for domain %(domain)s.\n"
"%(entry)s"
msgstr ""
"Se han encontrado varias coincidencias para el dominio %(domain)s.\n"
"%(entry)s"

msgid ""
"Host field should not be set on the instance until resources have been "
"claimed."
msgstr ""
"El campo de host no se debe establecer en la instancia hasta que los "
"recursos se hayan reclamado."

#, python-format
msgid ""
"Hostname %(hostname)s is longer than 63, truncate it to %(truncated_name)s"
msgstr ""
"La longitud del nombre de host %(hostname)s es más de 63, se trunca a "
"%(truncated_name)s"

msgid "Hypervisor driver does not support resume guests"
msgstr "El controlador de hipervisor no soporta reanudar invitados "

#, python-format
msgid "IP %s leased that is not associated"
msgstr "La IP %s alquilada que no está asociada "

#, python-format
msgid "IP %s released that is not associated"
msgstr "IP %s liberada que no está asociada"

#, python-format
msgid "IP %s released that was not leased"
msgstr "IP %s liberada que no está alquilada"

#, python-format
msgid "IP |%s| leased that isn't allocated"
msgstr "IP |%s| alquilada que no está asignada"

#, python-format
msgid "ISO creation tool '%s' does not exist."
msgstr "La herramienta de creación de ISO '%s' no esiste."

#, python-format
msgid ""
"Ignoring DiskNotFound exception while detaching volume %(volume_id)s from "
"%(mp)s: %(err)s"
msgstr ""
"Ignorando excepción DiskNotFound mientras se separa volumen %(volume_id)s de "
"%(mp)s: %(err)s"

#, python-format
msgid "Ignoring Volume Error on vol %(vol_id)s during delete %(exc)s"
msgstr ""
"Ignorando Error de volumen en volumen %(vol_id)s durante la remocion %(exc)s"

#, python-format
msgid "Ignoring error injecting %(inject)s into image (%(e)s)"
msgstr "Ignorando el error al inyectar %(inject)s en la imagen (%(e)s)"

#, python-format
msgid "Ignoring error injecting data into image %(image)s (%(e)s)"
msgstr "Ignorando el error al inyectar datos en la imagen %(image)s (%(e)s)"

#, python-format
msgid "Ignoring error while configuring instance with agent: %s"
msgstr ""
"Ignorando error durante la configuración de instancia con el agente: %s"

#, python-format
msgid "Ignoring exception %(exc)s when getting PBDs for %(sr_ref)s"
msgstr "Ignorando la excepción %(exc)s al obtener las PBD de %(sr_ref)s"

#, python-format
msgid "Ignoring exception %(exc)s when unplugging PBD %(pbd)s"
msgstr "Ignorar la excepción %(exc)s al desconectar PBD %(pbd)s"

#, python-format
msgid "Ignoring malformed capability '%s'. Format should be 'key:val'."
msgstr "Ignorando función incorrecta '%s'. El formato debe ser 'key:val'."

#, python-format
msgid "Ignoring the invalid elements of the option %(name)s: %(options)s"
msgstr "Ignorando los elementos inválidos de la opción %(name)s: %(options)s"

#, python-format
msgid "Ignoring volume cleanup failure due to %s"
msgstr "Ignorando la anomalía de limpieza de volumen debido a %s "

msgid "Image not found during snapshot"
msgstr "No se ha encontrado la imagen durante la instantánea"

msgid ""
"In vmwareapi:vmops:_destroy_instance, exception while deleting the VM "
"contents from the disk"
msgstr ""
"Ocurrió excepción en vmwareapi:vmops:_destroy_instance mientras se "
"eliminaban los contenidos de la VM del disco"

#, python-format
msgid ""
"In vmwareapi:vmops:_destroy_instance, got this exception while un-"
"registering the VM: %s"
msgstr ""
"En vmwareapi:vmops:_destroy_instance, se obtuvo esta excepción mientras se "
"removía el registro de VM: %s"

#, python-format
msgid ""
"Instance %(instance)s is using a backing file %(backing)s which does not "
"appear in the image service"
msgstr ""
"La instancia %(instance)s utiliza un archivo de respaldo %(backing)s que no "
"aparece en el servicio de la imagen"

#, python-format
msgid ""
"Instance %(uuid)s appears to not be owned by this host, but by %(host)s. "
"Startup processing is being skipped."
msgstr ""
"Parece que la instancia %(uuid)s no pertenece a este host, sino a  "
"%(host)s.  Se está omitiendo el proceso de inicio."

msgid "Instance build timed out. Set to error state."
msgstr ""
"La compilación de instancia ha excedido el tiempo de espera. Se ha estado en "
"estado erróneo. "

msgid "Instance disappeared during reboot"
msgstr "La instancia ha desaparecido durante el rearranque"

msgid "Instance disappeared while detaching a PCI device from it."
msgstr ""
"La instancia ha desaparecido mientras se removía el dispositivo PCI de ella."

msgid "Instance does not exist on backend"
msgstr "La instancia no existe en back-end"

#, python-format
msgid "Instance failed network setup (attempt %(attempt)d of %(attempts)d)"
msgstr ""
"Fallo de configuración de red de la instancia (intento %(attempt)d de "
"%(attempts)d)"

msgid "Instance has had its instance_type removed from the DB"
msgstr ""
"En la instancia se ha eliminado el tipo de instancia de la base de datos"

#, python-format
msgid ""
"Instance in transitional state (%(task_state)s) at start-up and power state "
"is (%(power_state)s), clearing task state"
msgstr ""
"Instancia en estado transicional (%(task_state)s) al arranque y estado de "
"energía es (%(power_state)s), limpiando el estado de la tarea"

#, python-format
msgid ""
"Instance in transitional state (%(task_state)s) at start-up and power state "
"is (%(power_state)s), clearing task state and unpausing the instance"
msgstr ""
"La instancia está en estado temporal (%(task_state)s)  en el inicio y el "
"estado de energía es (%(power_state)s), limpiando el estado de la tarea y "
"reanudando la instancia"

msgid "Instance is not (soft-)deleted."
msgstr "La instancia no se suprime (de forma no permanente). "

#, python-format
msgid ""
"Instance is not stopped. Calling the stop API. Current vm_state: "
"%(vm_state)s, current task_state: %(task_state)s, original DB power_state: "
"%(db_power_state)s, current VM power_state: %(vm_power_state)s"
msgstr ""
"La instancia no se ha detenido. Llamando a API detenido. vm_state actual: "
"%(vm_state)s, task_state actual: %(task_state)s, power_state de base de "
"datos original: %(db_power_state)s, power_state actual de VM: "
"%(vm_power_state)s"

msgid "Instance is paused unexpectedly. Ignore."
msgstr "La instancia se ha puesto en pausa inesperadamente. Ignorar. "

msgid "Instance is suspended unexpectedly. Calling the stop API."
msgstr ""
"La instancia se ha suspendido inesperadamente. Llamando a la API de "
"detención."

msgid "Instance is unexpectedly not found. Ignore."
msgstr "La instancia no se encuentra inesperadamente. Ignorar. "

msgid ""
"Instance lifecycle events from the compute driver have been disabled. Note "
"that lifecycle changes to an instance outside of the compute service will "
"not be synchronized automatically since the _sync_power_states periodic task "
"is also disabled."
msgstr ""
"Se han inhabilitado los eventos de ciclo de vida dela instancia del "
"controlador de cómputo. Tenga en cuenta que no se sincronizarán "
"automáticamente  los cambios de ciclo de vida en una instancia fuera del "
"servicio de cómputo ya que la tarea periódica  _sync_power_states también "
"está inhabilitada."

msgid "Instance may be still running, destroy it again."
msgstr "Puede que la instancia aún se esté ejecutando, vuelva a destruirla."

msgid "Instance not resizing, skipping migration."
msgstr "La instancia no se está redimensionando, se salta la migración."

#, python-format
msgid ""
"Instance shutdown by itself. Calling the stop API. Current vm_state: "
"%(vm_state)s, current task_state: %(task_state)s, original DB power_state: "
"%(db_power_state)s, current VM power_state: %(vm_power_state)s"
msgstr ""
"La instancia se apagó por si sola. Llamando a API detenido. vm_state actual: "
"%(vm_state)s, task_state actual: %(task_state)s, power_state de base de "
"datos original: %(db_power_state)s, power_state actual de VM: "
"%(vm_power_state)s"

msgid "Invalid 'agent_present' value. Falling back to the default."
msgstr ""
"Valor inválido 'agent_present'. Regresado hacia configuración predeterminada."

#, python-format
msgid "Invalid cachemode %(cache_mode)s specified for disk type %(disk_type)s."
msgstr ""
"Modalidad de caché %(cache_mode)s no válida especificada para el tipo de "
"disco %(disk_type)s."

#, python-format
msgid "Invalid value '%d' for image_compression_level"
msgstr "Valor inválido '%d' para image_compression_level"

#, python-format
msgid "Invalid value '%s' for torrent_images"
msgstr "valor inválido '%s' para torrent_images"

#, python-format
msgid "Invalid value for id mapping %s"
msgstr "Valor para mapeo de identificación no válido %s"

#, python-format
msgid "Key '%(key)s' not in instances '%(inst)s'"
msgstr "La clave '%(key)s' no está en las instancias '%(inst)s'"

msgid ""
"Libvirt module could not be loaded. NWFilterFirewall will not work correctly."
msgstr ""
"El módulo lLibvirt no se ha podido cargar. NWFilterFirewall no funcionará "
"correctamente."

#, python-format
msgid ""
"Lost connection to XenAPI during call to %(plugin)s.%(fn)s.  Retrying call."
msgstr ""
"Conexión perdida a XenAPI durante la llamada a %(plugin)s.%(fn)s. "
"Reintentando llamada."

#, python-format
msgid ""
"Metadata value %(value)s for %(key)s is not of type %(type)s. Using default "
"value %(default)s."
msgstr ""
"Valor de metadatos %(value)s para %(key)s no es de tipo %(type)s. Utilizando "
"valor por defecto %(default)s."

msgid "Migration operation was cancelled"
msgstr "Se ha cancelado la operación de migración"

#, python-format
msgid "Mountpoint cannot be translated: %s"
msgstr "Punto de montaje no puede ser traducido: %s"

#, python-format
msgid "Multiple base images for image: %s"
msgstr "Múltiple imágenes base para la imagen: %s"

#, python-format
msgid ""
"Network %(id)s not matched with the tenants network! The ports tenant "
"%(tenant_id)s will be used."
msgstr ""
"La red %(id)s no coincide con las redes de los inquilinos! El puerto del "
"inquilino %(tenant_id)s será utilizado."

#, python-format
msgid "Neutron error: Fixed IP %s is already in use."
msgstr "Error Neutron: La IP fija %s ya está en uso."

#, python-format
msgid ""
"Neutron error: MAC address %(mac)s is already in use on network %(network)s."
msgstr ""
"Error de Neutron: la dirección MAC %(mac)s ya está en uso en la red "
"%(network)s."

#, python-format
msgid "Neutron error: No more fixed IPs in network: %s"
msgstr "Error de Neutron: No hay más IPs fijas en red: %s"

#, python-format
msgid "Neutron error: Port quota exceeded in tenant: %s"
msgstr "Error de Neutron: capacidad de puerto excedida en usuario: %s"

#, python-format
msgid "No Network for FixedIP: %s"
msgstr "No hay red para la IP Fija : %s"

#, python-format
msgid "No VirtualInterface for FixedIP: %s"
msgstr "No hay Interfaz Virtual para la IP Fija : %s"

#, python-format
msgid "No cell_name for %(method)s() from API"
msgstr "No hay cell_name para %(method)s() desde la API"

msgid "No cell_name for instance update from API"
msgstr "No hay cell_name para actualización de instancia desde la API"

#, python-format
msgid "No compute node record for %(host)s:%(node)s"
msgstr "No hay registro de nodo de cómputo para %(host)s:%(node)s"

#, python-format
msgid "No compute service record found for host %(host)s"
msgstr "No hay registro de servicio de cómputo para host %(host)s"

msgid "No free nbd devices"
msgstr "No hay dispositivos nbd libres"

#, python-format
msgid ""
"No host name specified for the notification of HostAPI.%s and it will be "
"ignored"
msgstr ""
"No ha sido especificado un nombre de anfitrión para la notificación de "
"HostAPI.%s y será ignorada"

#, python-format
msgid "No match when trying to update BDM: %(bdm)s"
msgstr "No se encontró resultado al intentar actualizar BDM: %(bdm)s"

msgid "No more network or fixed IP to be allocated"
msgstr "No hay más redes o IP fijas para asignar"

msgid "No valid host found for rebuild"
msgstr "No se ha encontrado un host válido para reconstruir"

msgid "No valid host found for unshelve instance"
msgstr "No se ha encontrado anfitrión válido para extraer instancia"

#, python-format
msgid "Node %(uuid)s has a malformed \"%(prop)s\". It should be an integer."
msgstr "Nodo %(uuid)s tiene \"%(prop)s\" incorrectas. Debe ser un entero."

msgid ""
"Node field should not be set on the instance until resources have been "
"claimed."
msgstr ""
"El campo Nodo no debe ser establecido en la instancia hasta que los recursos "
"han sido reclamados."

#, python-format
msgid "Not deleting key %s"
msgstr "Sin eliminar la clave %s"

msgid ""
"Old property name \"hw_watchdog_action\" is now deprecated and will be "
"removed in the next release. Use updated property name \"hw:watchdog_action"
"\" instead"
msgstr ""
"Ahora se prohíbe el uso del nombre de propiedad antiguo \"hw_watchdog_action"
"\" y será eliminado en la siguiente versión. Utilizar en cambio nombre de "
"propiedad actualizado \"hw:watchdog_action\" "

msgid "Paused instance shutdown by itself. Calling the stop API."
msgstr ""
"La instancia pausada se ha apagado a si misma. Llamando la API de detención."

#, python-format
msgid "Periodic cleanup failed to delete instance: %s"
msgstr "Limpieza periódica falló al eliminar la instancia: %s"

#, python-format
msgid "Periodic reclaim failed to delete instance: %s"
msgstr "Reclamación periódica falló al eliminar instancia: %s"

#, python-format
msgid ""
"Periodic task is updating the host stat, it is trying to get disk "
"%(i_name)s, but access is denied. It is most likely due to a VM that exists "
"on the compute node but is not managed by Nova."
msgstr ""
"La tarea periódica está actualizando las estadísticas del host, está "
"intentando obtener disco %(i_name)s, pero el acceso es denegado. Es lo más "
"probable debido a una VM que existe en el nodo de cómputo, pero no es "
"gestionada por Nova."

#, python-format
msgid ""
"Periodic task is updating the host stat, it is trying to get disk "
"%(i_name)s, but disk file was removed by concurrent operations such as "
"resize."
msgstr ""
"La tarea periódica está actualizando las estadísticas del host, está "
"intentando obtener el disco %(i_name)s, pero el disco ha sido removido por "
"operaciones concurrentes como la modificación de tamaño."

#, python-format
msgid ""
"Periodic task is updating the host stats, it is trying to get disk info for "
"%(i_name)s, but the backing volume block device was removed by concurrent "
"operations such as resize. Error: %(error)s"
msgstr ""
"La tarea periódica está actualizando las estadísticas del host, está "
"intentando obtener el disco %(i_name)s, pero el dispositivo de bloque de "
"volumen de reserva fue eliminado por operaciones concurrentes como el "
"redimensionamiento. Error: %(error)s"

msgid ""
"Please consider to execute setup before trying to configure debug log "
"message."
msgstr ""
"Por favor considere ejecutar la configuración antes de intentar configurar "
"el mensaje de inicio de depuración."

#, python-format
msgid "Port %s does not exist"
msgstr "El puerto %s no existe"

#, python-format
msgid "Power state %s not found."
msgstr "No se ha encontrado estado de energía %s."

#, python-format
msgid "Quota exceeded for %s, tried to allocate floating IP"
msgstr "Cuota excedida para %s, intentando asignar dirección IP flotante"

#, python-format
msgid ""
"Quota exceeded for project %(pid)s, tried to allocate fixed IP. %(used)s of "
"%(allowed)s are in use or are already reserved."
msgstr ""
"Se ha superado la capacidad para %(pid)s, se ha intentado asignar IP fija. "
"%(used)s de %(allowed)s están en uso o ya están reservados."

msgid "Race detected in libvirt.virDomain.info, trying one more time"
msgstr ""
"Anticipación detectada en libvirt.virDomain.info, intnetando una vez más"

msgid "Reboot failed but instance is running"
msgstr "Ha fallado el reinicio pero la instancia se mantiene en ejecución"

msgid ""
"Reboot failed due to bad volumes, detaching bad volumes and starting halted "
"instance"
msgstr ""
"Se ha encontrado un error en el rearranque debido a volúmenes erróneos; se "
"van a desconectar los volúmenes erróneos e iniciar la instancia parada"

msgid "Rejecting snapshot request, snapshots currently disabled"
msgstr ""
"Rechazando solicitud de instantánea, instantáneas inhabilitadas actualmente"

msgid ""
"Relative blockrebase support was not detected. Continuing with old behaviour."
msgstr ""
"El soporte relativo a blockRebase no fue detectado. Continuando con el "
"antiguo comportamiento."

#, python-format
msgid ""
"Report interval must be less than service down time. Current config: "
"<service_down_time: %(service_down_time)s, report_interval: "
"%(report_interval)s>. Setting service_down_time to: %(new_service_down_time)s"
msgstr ""
"El intérvalo de reporte debe ser menor que el tiempo fuera de servicio. "
"Configuración actual: <service_down_time: %(service_down_time)s, "
"report_interval: %(report_interval)s>. Configuración service_down_time a: "
"%(new_service_down_time)s"

#, python-format
msgid "Retry SR scan due to error: %s"
msgstr "Reintentando escaneo de SR debido a error: %s"

#, python-format
msgid ""
"Retrying %(name)s after a MessagingTimeout, attempt %(attempt)s of "
"%(retries)s."
msgstr ""
"Intentando de nuevo %(name)s  después de una Desconexión de Mensaje, intento "
"%(attempt)s de %(retries)s."

#, python-format
msgid "Root disk file creation failed - %s"
msgstr "Fallo al crear el archivo del disco raíz - %s"

#, python-format
msgid ""
"Running Nova with a VMware vCenter version less than %(version)s is "
"deprecated. The required minimum version of vCenter will be raised to "
"%(version)s in the 13.0.0 release."
msgstr ""
"Se desaprueba la ejecución de Nova con una versión VMware vCenter inferior a "
"%(version)s. La versión mínima requerida de vCenter será aumentada a "
"%(version)s en el lanzamiento 13.0.0."

msgid ""
"Running libvirt-lxc without user namespaces is dangerous. Containers spawned "
"by Nova will be run as the host's root user. It is highly suggested that "
"user namespaces be used in a public or multi-tenant environment."
msgstr ""
"Ejecutar libvirt-lxc sin sin espacios de nombre de usuario es peligroso. Los "
"contenedores generados por Nova se ejecutarán como un usuario raíz del host. "
"Se sugiere firmemente usar espacios de nombre de usuario en un ambiente "
"público o multi-usuario."

msgid "Server with unsupported policy cannot be rebuilt"
msgstr "No se puede reconstruir servidor con política no soportada"

msgid "Service killed that has no database entry"
msgstr "Se detuvo un servicio sin entrada en la base de datos"

#, python-format
msgid "Setting instance to %s state."
msgstr "Estableciendo instancia hacia estado %s"

#, python-format
msgid "Setting migration %(migration_id)s to error: %(reason)s"
msgstr "Estableciendo la %(migration_id)s en error: %(reason)s"

#, python-format
msgid "Skipping detach because VBD for %s was not found"
msgstr "Omitiendo desasociación debido a que VBD para %s no ha sido encontrado"

#, python-format
msgid ""
"Subnet(s) too large, defaulting to /%s.  To override, specify network_size "
"flag."
msgstr ""
"Subred(es) demasiado grande(s), se usará el valor predeterminado /%s. Para "
"sustituirlo, especifique el distintivo network_size."

#, python-format
msgid ""
"Tenant_id %(tenant_id)s does not match tenant_id of instance %(instance_id)s."
msgstr ""
"Tenant_id %(tenant_id)s no coincide con tenant_id de la instancia "
"%(instance_id)s."

msgid ""
"This key manager is insecure and is not recommended for production "
"deployments"
msgstr ""
"Este administrador de claves es inseguro y no se recomienda para "
"liberaciones de producción."

msgid "This shouldn't be getting called except during testing."
msgstr "Esto no se debe llamar excepto durante las pruebas. "

msgid ""
"Timed out waiting for nova-conductor.  Is it running? Or did this service "
"start before nova-conductor?  Reattempting establishment of nova-conductor "
"connection..."
msgstr ""
"Se ha excedido el tiempo de espera de nova-conductor. ¿Se está ejecutando? "
"¿O este servicio se ha iniciado antes de nova-conductor? Intentando "
"establecer de nuevo la conexión nova-conductor..."

msgid "Timed out while waiting for soft shutdown."
msgstr "En reposo mientras se espera por desconexión."

#, python-format
msgid "Timeout waiting for vif plugging callback for instance %(uuid)s"
msgstr ""
"Tiempo excedido para la llamada inversa de la conexión vif para la instancia "
"%(uuid)s"

msgid "Too many id maps, only included first five."
msgstr "Demasiados mapas de IP, incluir únicamente los primeros cinco."

#, python-format
msgid ""
"Treating negative config value (%(retries)s) for 'block_device_retries' as 0."
msgstr ""
"Tratando como 0 el valor negativo de configuración (%(retries)s) para "
"'block_device_retries' ."

#, python-format
msgid ""
"Treating negative config value (%(retries)s) for 'glance.num_retries' as 0."
msgstr ""
"Tratando como 0 el valor negativo de configuración (%(retries)s) para "
"'glance.num_retries' ."

#, python-format
msgid ""
"Tried to remove rule that was not there: %(chain)r %(rule)r %(wrap)r %(top)r"
msgstr ""
"Se ha intentado eliminar una regla que no estaba allí: %(chain)r %(rule)r "
"%(wrap)r %(top)r"

#, python-format
msgid ""
"Trying to remove device with %(status)s ownership %(instance_uuid)s because "
"of %(pci_exception)s"
msgstr ""
"Intentando remover dispositivo con %(status)s con propiedad "
"%(instance_uuid)s debido a %(pci_exception)s"

#, python-format
msgid "URI %(uri)s does not support connection events: %(error)s"
msgstr "URI %(uri)s no soporta eventos de conexión: %(error)s"

#, python-format
msgid "URI %(uri)s does not support events: %(error)s"
msgstr "URI %(uri)s no soporta eventos: %(error)s"

#, python-format
msgid "URI %(uri)s does not support full set of host capabilities: %(error)s"
msgstr ""
"URI %(uri)s no soporta el conjunto completo de funciones del host: %(error)s"

#, python-format
msgid "URI %(uri)s does not support listDevices: %(error)s"
msgstr "URI %(uri)s no soporta listDevices: %(error)s"

#, python-format
msgid "Unable to delete %(file)s. Exception: %(ex)s"
msgstr "Incapaz de remover %(file)s. Excepción: %(ex)s"

#, python-format
msgid "Unable to dequalify.  %(name)s is not in %(domain)s.\n"
msgstr "No se puede descalificar.  %(name)s no está en %(domain)s.\n"

msgid ""
"Unable to find a different image to use for rescue VM, using instance's "
"current image"
msgstr ""
"No se ha podido encontrar una imagen diferente para utilizarla para VM de "
"rescate, se utiliza la imagen actual de la instancia"

#, python-format
msgid ""
"Unable to find network matching '%(network_name)s', user will have to enter "
"IP manually..."
msgstr ""
"Incapaz de encontrar red coincidente '%(network_name)s', el usuario deberá "
"introducir una dirección IP manualmente..."

#, python-format
msgid "Unable to force TCG mode, libguestfs too old? %s"
msgstr "No se puede forzar el modo TCG ¿libguestfs es demasiado antiguo? %s"

#, python-format
msgid "Unable to mount image %(image)s with error %(error)s. Cannot resize."
msgstr ""
"No se puede montar la imagen %(image)s con error %(error)s. No se puede "
"redimensionar."

msgid "Unable to update metadata, VM not found."
msgstr "Incapaz de actualizar metadatos, la VM no ha sido encontrada."

#, python-format
msgid "Unable to update the agent due to: %(exc)s"
msgstr "Incapaz de actualiar el agente debido a: %(exc)s"

#, python-format
msgid ""
"Unexpected confirmation status '%(status)s' of migration %(id)s, exit "
"confirmation process"
msgstr ""
"Estado de confirmación inesperado '%(status)s' de la migración %(id)s, salir "
"del proceso de confirmación"

#, python-format
msgid "Unexpected migration job type: %d"
msgstr "Tipo de trabajo de migración inesperado: %d"

#, python-format
msgid "Unexpected power state %d"
msgstr "Estado de alimentación inesperado %d"

#, python-format
msgid "Unknown base file: %s"
msgstr "Archivo de base desconocido: %s "

#, python-format
msgid "Unknown content in connection_info/qos_specs: %s"
msgstr "Contenido desconocido en connection_info/qos_specs: %s"

#, python-format
msgid "Unrecognized cache_images value '%s', defaulting to True"
msgstr ""
"Valor cache_images no reconocido '%s', se toma True como valor predeterminado"

msgid "VCPUs not set; assuming CPU collection broken"
msgstr ""
"VCPU no establecidas; suponiendo que la colección de CPU se ha interrumpido"

msgid "VM already halted, skipping shutdown..."
msgstr "VM ya se ha detenido, omitiendo la conclusión... "

msgid "VM is not present, skipping destroy..."
msgstr "VM no está presente, omitiendo destrucción... "

msgid "VM is not present, skipping soft delete..."
msgstr "VM no está presente, omitiendo supresión no permanente... "

msgid ""
"Value specified in conf file for xenserver.torrent_base_url does not contain "
"a slash character, therefore it will not be used as part of the torrent URL. "
"Specify a valid base URL as defined by RFC 1808 (see step 6)."
msgstr ""
"El valor especificado en el  archivo de configuración para xenserver."
"torrent_base_url no contiene un slash, por lo tanto, no se utilzará como "
"parte de la URL de torrent . Especifique una URL base como lo define  RFC "
"1808 (ver paso 6)."

#, python-format
msgid "Volume %s should be encrypted but there is no encryption metadata."
msgstr ""
"El volumen %s debe estar cifrado pero no hay metadatos de encriptación."

#, python-format
msgid ""
"Volume id: %(vol_id)s finished being created but its status is "
"%(vol_status)s."
msgstr ""
"Volumen identificado como: %(vol_id)s ha terminado de crearse pero su estado "
"es %(vol_status)s."

msgid ""
"WARNING: Using the remote debug option changes how Nova uses the eventlet "
"library to support async IO. This could result in failures that do not occur "
"under normal operation. Use at your own risk."
msgstr ""
"ADVERTENCIA: Usar la opción de depuración remota cambia la forma en que Nova "
"utiliza la librería Eventlet para soportar IO asíncrono. Esto puede resultar "
"en fallos que no ocurrirían durante una operación normal. Úsela bajo su "
"propio riesgo."

#, python-format
msgid "Warning, memory usage is 0 for %(instance)s on baremetal node %(node)s."
msgstr ""
"Advertencia, el uso de memoria es 0 para %(instance)s en nodo nativo "
"%(node)s."

#, python-format
msgid ""
"Warning, number of cpus is 0 for %(instance)s on baremetal node %(node)s."
msgstr ""
"Advertencia, el número de cpus es 0 para %(instance)s en nodo nativo "
"%(node)s."

#, python-format
msgid ""
"While synchronizing instance power states, found %(num_db_instances)s "
"instances in the database and %(num_vm_instances)s instances on the "
"hypervisor."
msgstr ""
"Se encontraron instancias %(num_db_instances)s en la base de datos e "
"instancias %(num_vm_instances)s en el hipervisor mientras de sincronizaban "
"los estados de energía de la instancia. "

msgid ""
"X-Instance-ID present in request headers. The 'service_metadata_proxy' "
"option must be enabled to process this header."
msgstr ""
"X-Instance-ID presente en cabecera requerida. Se debe habilitar la opción "
"'service_metadata_proxy' para procesar esta cabecera"

#, python-format
msgid ""
"X-Instance-ID-Signature: %(signature)s does not match the expected value: "
"%(expected_signature)s for id: %(requestor_id)s. Request From: "
"%(requestor_address)s"
msgstr ""
"X-Instance-ID-Signature: %(signature)s no coincide con el valor esperado: "
"%(expected_signature)s para id: %(requestor_id)s. Solicitud de: "
"%(requestor_address)s"

#, python-format
msgid "_migrate_disk_resizing_up failed to rollback: %s"
msgstr "_migrate_disk_rezising_up fallido al revertir: %s"

#, python-format
msgid ""
"attach interface failed , try to deallocate port %(port_id)s, reason: %(msg)s"
msgstr ""
"Ha fallado la conexión de interfaz, intente desasignar puerto  %(port_id)s, "
"razón: %(msg)s"

#, python-format
msgid "couldn't obtain the memory from domain: %(uuid)s, exception: %(ex)s"
msgstr "no se pudo obtener la memoria del dominio: %(uuid)s, excepción: %(ex)s"

#, python-format
msgid "cpu_arch not defined for node '%s'"
msgstr "cpu_arch no está definido por nodo '%s'"

msgid "datastore_regex is ignored when PBM is enabled"
msgstr "Se ignora datastore_regex cuando PBM se habilita"

#, python-format
msgid "deallocate port %(port_id)s failed"
msgstr "Ha fallado la designación del puerto %(port_id)s"

#, python-format
msgid "hostname %s too long, truncating."
msgstr "el nombre de host %s es muy largo, acortando."

#, python-format
msgid "ignoring missing logical volume %(path)s"
msgstr "Ignorando la falta de volumen lógico %(path)s"

#, python-format
msgid ""
"image %(id)s at (%(base_file)s): warning -- an absent base file is in use! "
"instances: %(instance_list)s"
msgstr ""
"%(id)s (%(base_file)s): aviso -- se está utilizando un archivo base ausente. "
"instancias: %(instance_list)s"

#, python-format
msgid "instance's host %s is down, deleting from database"
msgstr ""
"el host de la instancia %s está inactivos, se suprime de la base de datos"

#, python-format
msgid ""
"instance-dns-zone is |%(domain)s|, which is in availability zone |%(zone)s|. "
"Instance is in zone |%(zone2)s|. No DNS record will be created."
msgstr ""
"instance-dns-zone es |%(domain)s|, que está en la zona de disponibilidad |"
"%(zone)s|. La instancia está en la zona |%(zone2)s|. No se creará ningún "
"registro de DNS."

#, python-format
msgid "instance-dns-zone not found |%s|."
msgstr "instance-dns-zone no encontrada |%s|"

msgid "ipxe_boot is True but no ISO image found"
msgstr "ipxe_boot establecido en True pero no se ha encontrado imagen ISO"

msgid "ipxe_boot_menu_url not set, user will have to enter URL manually..."
msgstr ""
"ipxe_boot_menu_url no establecido, el usuario debe ingresar la URL "
"manualmente..."

msgid "ipxe_network_name not set, user will have to enter IP manually..."
msgstr ""
"ipxe_network_name no establecido, el usuario debe ingresar la dirección IP "
"manualmente..."

#, python-format
msgid "isLuks exited abnormally (status %(exit_code)s): %(stderr)s"
msgstr "isLuks ha salido de manera anormal (estado%(exit_code)s): %(stderr)s"

#, python-format
msgid "multiple fixed_ips exist, using the first IPv4 fixed_ip: %s"
msgstr "existen múltiples fixed_ips, utilizando la primer IPv4 fixed_ip: %s"

#, python-format
msgid ""
"my_ip address (%(my_ip)s) was not found on any of the interfaces: %(ifaces)s"
msgstr ""
"No se encontró la dirección my_ip (%(my_ip)s) en ningna de las interfaces: "
"%(ifaces)s"

#, python-format
msgid "plug_vifs() failed %(cnt)d. Retry up to %(max_retry)d."
msgstr "plug_vifs() ha fallado %(cnt)d. Intentando hasta %(max_retry)d."

#, python-format
msgid "rbd remove %(volume)s in pool %(pool)s failed"
msgstr "Falló la eliminación rbd %(volume)s en pool %(pool)s "

msgid "set_admin_password is not implemented by this driver or guest instance."
msgstr ""
"esta instancia de invitado o controlador no implementa set_admin_password ."

#, python-format
msgid ""
"trying to inject a file into a non-running (state: %(current_state)s "
"expected: %(expected_state)s)"
msgstr ""
"intentando inyectar un archivo hacia un inactivo (estado: %(current_state)s "
"esperado: %(expected_state)s)"

#, python-format
msgid ""
"trying to reboot a non-running instance: (state: %(state)s expected: "
"%(running)s)"
msgstr ""
"intentando rearrancar una instancia que no se está ejecutando: (estado: "
"%(state)s se esperaba: %(running)s)"

#, python-format
msgid ""
"trying to snapshot a non-running instance: (state: %(state)s expected: "
"%(running)s)"
msgstr ""
"intentando hacer una instantánea de una instancia que no se está ejecutando: "
"(estado: %(state)s se esperaba: %(running)s)"

#, python-format
msgid ""
"vmware_ostype from image is not recognised: '%(ostype)s'. An invalid os type "
"may be one cause of this instance creation failure"
msgstr ""
"no se reconoce vmware_ostype de la imagen: '%(ostype)s'. Una de las causas "
"de esta falla en la creación de esta instancia puede ser un tipo de sistema "
"operativo no válido"
